templates/zerver/integrations/git.md

```diff
@@ -1,31 +1,57 @@
-Get Zulip notifications for your Git repositories!
+# Zulip Git integration
+
+Get Zulip notifications for pushes to your Git repositories!
+
+!!! tip ""
+
+    Zulip offers integrations for
+    [various repository hosting services][other-integrations], and also
+    supports this direct Git integration for unsupported services.
```

Comment:
The tip format looks a little odd in the rendered documentation here. I'd maybe just expand the content of the  integration description ...

> Get Zulip notifications for pushes to your Git repositories!
>
> Note that Zulip also has integrations with [GitHub](https://zulip.com/integrations/doc/github), [GitLab](https://zulip.com/integrations/doc/gitlab), and various other [version control hosting services](https://zulip.com/integrations/version-control), which do not require access to the Zulip server to set up.